### ISSUE TITLE
Add a "feature" field

### ISSUE THREAD

**Issue by babolivier** [cozy-labs/cozy-registry#29]
It could be interesting for apps to fill a "feature" field describing with one keywoard the app's main feature ("irc" for TheLounge, "pastebin" for Haste, and so on), in case we have several apps providing the same service.

Let's imagine for example that we have two or more apps providing the "irc" feature. We could imagine that the user would be able to define a default IRC client (or Cozy could select the first/last installed app with this feature as a default), and add a conditionnal to the IRC support link on the help panel so it opens that client instead of Freenode's webchat.

We can observe a similar behaviour in Archlinux's packages structure, where a package can use a "provides" field. For example, the AUR package named "nodejs-lts-bin", which provides Node.js v4 (the one packaged in the official repositories is the latest, v6), can tell via this field pacman (the package manager) that it "provides" the programm "nodejs", so every other package depending on Node.js will use this one if installed instead of grabbing the official "nodejs" package from the official repositories.

Suggested by @Zykino on IRC.

**Comment by Zykino** [comment]
In the IRC nicofrand pointed out that link describing the same behavior done in Firefox
https://developer.mozilla.org/fr/docs/Gestionnaires_de_protocoles_web

**Comment by babolivier** [comment]
@Zykino Yup, that can be used in this specific use case because IRC uses a [standardized](https://tools.ietf.org/html/rfc1459) protocol, but that's not always the case. For example, there's no protocol for calling a file browser or a bookmarks manager, at least not that I'm aware of.
Plus I'm not sure TheLounge would be ready to support this kind of calls in its current state.
